src/app/clusters/ota-requestor/ExtendedOTARequestorDriver.cpp

```diff
@@ -49,7 +49,6 @@ void ExtendedOTARequestorDriver::UpdateAvailable(const UpdateDescription & updat
         if (err != CHIP_NO_ERROR)
         {
             ChipLogError(SoftwareUpdate, "Failed to get user consent subject");
-            HandleError(UpdateFailureState::kDelayedOnUserConsent, err);
```

Comment:
Why did this get removed?

Reply:
I originally removed `HandleError()` when it was an empty function and did nothing via commit bb7d9ffe5dcf3ccc8229a9fc0f9fd725d2bc9400.  I'll add this call back.